torch/_torch_docs.py

```diff
@@ -3976,6 +3976,26 @@ def merge_dicts(*dicts):
     tensor([-0.2018, -0.2962, -0.0821, -1.1831])
 """.format(**single_dim_common))
 
+add_docstr(torch.promote_types,
+           r"""
+promote_types(type1, type2) -> dtype
+
+Returns the :class:`torch.dtype` with the smallest size and scalar kind to which
+`type1` and `type2` may be safely cast. See type promotion :ref:`documentation <type-promotion-doc>`
```

Comment:
"safely cast" isn't really defined here and I would argue are casting rules aren't really safe (i.e. int32 + float32) -> float32 can lose precision.  Can you reword this somehow?

Reply:
updated to a more specific explanation.